2197/2/helper.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'artist'
+require_relative 'constants'
+
+# This is a module to decide error in reek
+module Helper
```

Comment:
instead of including this module everywhere - it's better to convert this to service class

Reply:
Why everywhere? I include it only once, don't understand that